src/components/constants/testdata.json

```diff
@@ -0,0 +1,728 @@
+[
```

Comment:
Rename this file as - something like test_packets or whatever. `testdata.json` is too  generic and better to be descriptive about what kind of test data is contained by calling appropriate filename. So that later we can include multiple testdatas.